### ISSUE TITLE
Repo maintenance

### ISSUE THREAD

**Issue by sandstrom** [t1m0n/air-datepicker#558]
A few smaller suggestions to make this excellent library easier to maintain.

### Github repo suggestions

- Disable the wiki tab, not used
- Disable the projects tab, not used
- Enable the discussions tab (good for asking questions)
- For issues that aren't real issues, or doesn't have a clear explanation with steps to reproduce and a failing test, just move them all over to the discussions tab. That way, the list of open issues can be much lower, less noise.
  - Here are a few examples of issues that should be moved to a discussions tab:
    - https://github.com/t1m0n/air-datepicker/issues/419
    - https://github.com/t1m0n/air-datepicker/issues/417
    - https://github.com/t1m0n/air-datepicker/issues/429
- I'd replace the "Recent updates" part of the readme with a link to the changelog file. That way, you don't have to maintain both lists, and it doesn't risk getting outdated.

### API/functionality suggestions

- Don't build too much functionality into the picker. It should also be reasonable to maintain. An easy rule is that if someone asks for something, ask them to put it as a feature suggestion under the 'Discussions' tab. Then only act on stuff that have at least a handful of upvotes, or more.
- Extension points are a great alternative to features, that way users can customize as needed themselves.
  - For example, the events that already exist is a great example of this (onSelect etc). Easy to maintain and can support a lot of customization, without having to build it into core.

### Bring more maintainers onboard

- It's always easier when others can help out. If someone has written good PRs or helped out in other ways. Ask them to come on board as a maintainer. You don't have to give them full admin rights immediately, and be somewhat cautions about giving out npm access. But having more maintainers on a project will make it much easier to maintain over time.

**Comment by t1m0n** [comment]
@sandstrom Thank you for your advices! 

Actually I didn't want to enable discussion tab, because don't have much time to answer all questions, but now I think it could be could idea.

I try to make datepicker more flexible and as a rule I don't add narrow or specific functionality, on the contrary I try to extend existing events and options to be able to solve complex cases. 

My goal is to keep things simple and small sized)

**Comment by sandstrom** [comment]
@t1m0n > I didn't want to enable discussion tab, because don't have much time to answer all questions

Totally get this! If you enable it, *don't spend time reading or answering any questions*. Don't even look at the tab, just let it be there, and use the 'move to discussion' button for issues that should really be a Q&A question, just to get rid of them.

Let others in the community answer questions if they want (some will).

Basically it's just a place to dump things, so that you have fewer issues open.

**Comment by sandstrom** [comment]
@t1m0n Let me know if you want some help cleaning out irrelevant issues (stuff that's questions rather than bugs).
